clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/OperationsService.java

```diff
@@ -180,8 +180,7 @@ public OperationsService(
                           allowedAccountValidators.forEach(
                               it -> it.validate(username, allowedAccounts, description, errors));
 
-                          // TODO(rz): Assert `description` is T
-                          descriptionAuthorizer.authorize(description, errors);
+                          descriptionAuthorizers.forEach(it -> it.authorize(description, errors));
```

Comment:
It would be nice if a `DescriptionAuthorizer` had a better means of indicating successful auth, rather than just mutating the passed in `Errors` object.

_or_

Perhaps this changes to `findFirst(it.supports(description)).authorize(description, errors)`. For this to work, we'd need to mark the existing `DefaultDescriptionAuthorizer` with the lowest priority.

That would avoid a need to explicitly clean any errors up?

Reply:
I like the findFirst approach

Reply:
Yeah.  findFirst sounds good to me :+1: